x/foundation/keeper/keys.go

```diff
@@ -19,7 +19,8 @@ var (
 	proposalByVPEndKeyPrefix = []byte{0x13}
 	voteKeyPrefix            = []byte{0x14}
 
-	grantKeyPrefix = []byte{0x20}
+	censorshipKeyPrefix = []byte{0x20}
```

Comment:
I'm just wondering conflict between `censorshipKeyPrefix=0x20` and `grantKeyPrefix=0x20`(previously used).
Is there any possibility that old grant data with grantKeyPrefix(0x20) gets loaded and failed to interpret that as censorship data?

Reply:
Technically, you are right about it. However, AFAIK, this sdk is only used by our nodes, and we won't simply change the binary. It would not allowed to apply this kind of change after the release, but it seems OK now.

Reply:
Ok, I understand